setup.py

```diff
@@ -52,6 +52,7 @@ def find_version(*file_paths):
         "docker>=4.3.1",
         "ordered-set>=4.0.2",
         "cfn-lint>=0.43.0",
+        "cfn_flip>=1.2.3",
```

Comment:
Is this an Amazon approved cfn flip? If it is the same as https://cfnflip.com/, we might have to switch to an internal tool.

Reply:
It's [this](https://github.com/awslabs/aws-cfn-template-flip) which I believe is maintained by AWS.